CDTDatastore.podspec

```diff
@@ -29,27 +29,52 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '6.0'
   s.osx.deployment_target = '10.8'
+
   s.requires_arc = true
 
-  s.source_files = 'Classes/**/*.{h,m}'
-  s.exclude_files = 'Classes/vendor/MYUtilities/*.{h,m}'
+  s.default_subspec = 'common'
+
+  s.subspec 'common' do |sp|
+    sp.prefix_header_contents = '#import "CollectionUtils.h"', '#import "Logging.h"', '#import "Test.h"'
+
+    sp.source_files = 'Classes/**/*.{h,m}'
+    sp.exclude_files = 'Classes/vendor/MYUtilities/*.{h,m}'
+    
+    sp.ios.exclude_files = 'Classes/osx'
+    sp.osx.exclude_files = 'Classes/ios'
+
+    sp.frameworks = 'SystemConfiguration'
 
-  s.prefix_header_contents = '#import "CollectionUtils.h"', '#import "Logging.h"', '#import "Test.h"'
+    sp.dependency 'CDTDatastore/no-arc'
+    sp.dependency 'CocoaLumberjack', '= 2.0.0-rc'
 
-  s.ios.exclude_files = 'Classes/osx'
-  s.osx.exclude_files = 'Classes/ios'
-  # s.public_header_files = 'Classes/common/CloudantSync.h'
+    sp.default_subspec = 'SQLite'
 
-  s.dependency 'FMDB', '= 2.3'
-  s.dependency 'CocoaLumberjack', '= 2.0.0-rc'
+    sp.subspec 'SQLite' do |ssp|
+      ssp.library = 'sqlite3', 'z'
 
-  s.frameworks = 'SystemConfiguration'
-  s.library = 'sqlite3', 'z'
+      ssp.dependency 'FMDB', '= 2.3'
+    end
+
+    sp.subspec 'SQLCipher' do |ssp|
+      ssp.library = 'z'
+
+      ssp.dependency 'FMDB/SQLCipher', '= 2.3'
+    end
```

Comment:
This looks to me like you are getting the SQLCipher stuff by default, not optionally. I would like the default version to not pull in the SQLCipher dependency.